### ISSUE TITLE
Question: What happens to the db if I correct the metadata on my collection

### ISSUE THREAD

**Issue by khers** [sentriz/gonic#513]
gonic version: 0.16.4

if from docker, docker tag: latest
if from source, git tag/branch:

My collect metadata is a mess and this is leading to difficulty with a downstream project I work on. So I want to use a tool like beets to try and correct the metadata on the collection as well as the directory structure for same. My specific question is does Gonic prune entries from the data base when the file that is responsible for them is no longer present? For example, I have something like:
`media/Adderly, Cannonball/Cannonball Adderly/01 - intro.mp3`
But the metadata for this track has the artist listed as 'Cannonball Adderly'. I also have the opposite situation (where the directory on disk is 'correct' but the ID3 field is not). Will the extraneous  Artist entries be removed?

 More generally, how will the service handle the changes to the underlying media structure?

**Comment by sentriz** [comment]
yeah any bit of metadata that is no longer present in files should be removed. did you try it? feel free to reopen if not working
